test/test_clistyle.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 """Test the mycli.clistyle module."""
+import pytest
 
 from pygments.style import Style
 from pygments.token import Token
 
 from mycli.clistyle import style_factory
 
 
+@pytest.mark.skip(reason="incompatible with new prompt toolkit")
```

Comment:
@tsroten do we still want these tests? I don't know how to do these tests in promt toolkit 2.0.

Reply:
@meeuw I don't think we need these tests any longer. Thanks for noticing that!